geth/common/rpccall.go

```diff
@@ -94,23 +94,23 @@ func (r RPCCall) ParseValue() *hexutil.Big {
 
 // ParseGas returns the hex big associated with the call.
 // nolint: dupl
-func (r RPCCall) ParseGas() *hexutil.Big {
+func (r RPCCall) ParseGas() hexutil.Uint64 {
```

Comment:
Does it make any sense to actually return also an error here, what do you think?